openstack/evs/extensions/services/testing/requests_test.go

```diff
@@ -0,0 +1,26 @@
+package testing
+
+import (
+	"testing"
+
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/evs/extensions/services"
+
+	"github.com/opentelekomcloud/gophertelekomcloud/testhelper"
+	"github.com/opentelekomcloud/gophertelekomcloud/testhelper/client"
+)
+
+func TestListServices(t *testing.T) {
```

Comment:
What exact checking this test?

Reply:
It just tests whether the method works or not with mock data.